proprietary/Components/src/denhaag/inline-carousel.tokens.json

```diff
@@ -51,6 +51,12 @@
         "justify-content": { "value": "space-between" },
         "align-items": { "value": "center" },
         "max-width": { "value": "100%" },
+        "opacity": { "value": "0" },
```

Comment:
"opacity" hoeft van mij echt niet als token als je 'm overschrijft in de code. Scheelt overhead.

Reply:
Moeten we niet alles in tokens zetten?